library/pkcs5.c

```diff
@@ -222,7 +223,17 @@ int mbedtls_pkcs5_pbes2( const mbedtls_asn1_buf *pbe_params, int mode,
 
     if( ( ret = mbedtls_cipher_crypt( &cipher_ctx, iv, enc_scheme_params.len,
                               data, datalen, output, &olen ) ) != 0 )
-        ret = MBEDTLS_ERR_PKCS5_PASSWORD_MISMATCH;
+    {
+        /*
+         * PKCS5 supports DES or RC2. RC2 is not supported by Mbed TLS.
```

Comment:
Since PKCS5 now supports AES(but we still don't), I will add AES.
I don't think that adding other errors that we don't test is good. The code is already becoming difficult to read, since we don't have one common `FEATURE_UNAVAILABLE` error

Reply:
Are we still ruling out adding a `common_errors.h`, by the way? It would make so much sense, the only problem being that we can't replace existing errors so our choice is between piling on bad error codes after bad or having inconsistently sometimes-unified-sometimes-not error codes.

Reply:
I would really want to have one generic error for feature unavailable, It would make the code much cleaner.
However, this is still not approved, and not discussed yet